ci/scripts/create_instance.sh

```diff
@@ -77,6 +73,20 @@ GCP_NETWORK=${GCP_NETWORK##*/}
 GCP_SUBNETWORK=$(echo ${NETWORK_INTERFACE_INFO} | jq -r '.networkInterfaces[0].subnetwork')
 GCP_SUBNETWORK=${GCP_SUBNETWORK##*/}
 
+# Determine and store our attempt number
+cp old/attempts new/
+echo attempt >> new/attempts
+attempts=$(cat new/attempts | wc -l)
+echo $attempts > /tmp/retry_number
+
+if [ $attempts -eq 1 ]; then
+  ZONE=${MY_ZONE}
+else
+  PERMITTED_ZONES=(us-central1-a us-central1-b us-central1-c us-central1-f)
```

Comment:
if possible declare constants at the top or in a block somewhere, rather than inside an `if`?

Reply:
`MY_ZONE` comes from a `gcloud compute` query, so we can't really set these vars at the top. The zone vars are as grouped as I think makes sense.